core/app/[locale]/(default)/account/wishlists/_actions/change-wishlist-visibility.ts

```diff
@@ -73,28 +73,22 @@ export async function toggleWishlistVisibility(
     // eslint-disable-next-line no-console
     console.error(error);
 
-    if (error instanceof BigCommerceGQLError) {
-      return {
-        ...prevState,
-        lastResult: submission.reply({ formErrors: [t('Errors.unexpected')] }),
-        errorMessage: error.message.includes('Please sign in')
-          ? t('Errors.unauthorized')
-          : t('Errors.unexpected'),
-      };
-    }
+    if (error instanceof BigCommerceAuthError) {
```

Comment:
Are we just expecting an error of auth to be present here? GQL will never return a custom error?

Reply:
Correct. Changing the wishlist visibility is an action that requires 0 user input, so the only point of failures are auth, or an internal error